src/test/java/com/yelp/nrtsearch/server/luceneserver/search/collectors/CollectorTest.java

```diff
@@ -395,8 +395,12 @@ public void testCustomCollector() throws InvalidProtocolBufferException {
             "7", 93.0,
             "8", 92.0);
     Map<String, Double> collectedHits =
-        collectorsResult.get(COLLECTOR_NAME).getAnyResult().unpack(SearchResponse.class)
-            .getHitsList().stream()
+        collectorsResult
+            .get(COLLECTOR_NAME)
```

Comment:
It looks like the formatter wants `toBuilder()` on the same line but any other function call in a chain of calls on a different line? Kinda confusing, but I guess we can live with it if it can't be changed.

Reply:
Hmm, I guess this is just what spotless does now